openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -480,6 +487,7 @@ a[href].link-text-grey,
 // breadcrumb
 
 .breadcrumb-item {
+  max-width: 100%;
```

Comment:
Do we want this to be applied to all the breadcrumb?

Reply:
yes, so the case was a breadcrumb item was overflowing due the continuous string in it present. so this max-width will only used the width of the container and the split to the next line.